test/server.test.js

```diff
@@ -32,6 +33,9 @@ injectGlobal`
     display: flex;
     justify-content: center;
   }
+  .should-be-included {
+    content: 'css-foo matches regex and this rule is swallowed (current bug)'
+  }
 `
```

Comment:
Oh man. My bad on that. It looks like you are correct. @mitchellhamilton you mind taking one more look at this?